services/rfq/relayer/relconfig/getters.go

```diff
@@ -298,6 +298,43 @@ func (c Config) GetQuoteOffsetBps(chainID int, tokenName string, isOrigin bool)
 	return offset, nil
 }
 
+const defaultMaxBalance = 0
```

Comment:
Should we use something that resembles "positive infinity" here instead as a default value? This way we could actually use maxBalance = 0 to effectively stop quoting "to chain", while still quoting the other direction if that makes sense.

Reply:
Good idea, will use `nil` for this